voice/bxml/callbacks/transferAnswer.md

```diff
@@ -1,19 +1,38 @@
 {% method %}
 ##  Transfer Answer Event – <Transfer> verb
 When processing a [`<Transfer>`](../verbs/transfer.md) verb, this event is sent when a called party (B-leg) answers.  The event is sent to
-  the endpoint specified in the `transferAnswerUrl` attribute of the `<PhoneNumber>` tag that answered.  [`<PlayAudio>`](../verbs/playAudio.md) and/or [`<SpeakSentence>`](../verbs/speakSentence.md) verbs returned by this callback will be
+  the endpoint specified in the `transferAnswerUrl` attribute of the `<PhoneNumber>` tag that answered. Accepted verbs returned by this callback will be
```

Comment:
I would remove the word "accepted". The whole BXML document either gets executed or rejected. As written, it sounds kind of like allowed verbs will be executed and bad ones will be filtered out